lib/ethon/curls/options.rb

```diff
@@ -251,6 +251,7 @@ def self.option_type(type)
       option :easy, :opensocketdata, :cbdata, 164
       option :easy, :closesocketfunction, :callback, 208
       option :easy, :closesocketdata, :cbdata, 209
+      option :easy, :pathasis, :int, 234
```

Comment:
```suggestion
      option :easy, :path_as_is, :bool, 234
```

I think with underscores it's way easier to understand and the original option has the underscores as well. Please also make use type :bool as it's a bool flag like verbose (only 0 and 1 are valid params according to the linked docs).